__tests__/app.test.js

```diff
@@ -35,6 +36,34 @@ describe("GET /api/topics", () => {
   });
 });
 
+describe("GET /api/articles/:article_id", () => {
+  test("200: Responds with an article object corresponding to the id in the endpoint", () => {
+    return request(app)
+      .get("/api/articles/3")
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toMatchObject({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: expect.any(Number),
+          body: expect.any(String),
+          topic: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+        });
+      });
+  });
+  test("400: Responds with a bad request status when given an invalid id", () => {
+    return request(app)
+      .get("/api/articles/999")
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("bad request");
+      });
+  });
```

Comment:
Nice start to error handling, but this isnt the right error code for this kind of an endpoint. Have a look at the following notes on what kind of error handling we should be expecting for this kind endpoint: 

https://l2c.northcoders.com/courses/sd-notes/back-end#sectionId=error-handling,step=intro